build.gradle

```diff
@@ -18,46 +24,85 @@ repositories {
     jcenter()
 }
 
+publishing {
+    publications {
+        customLibrary(MavenPublication) {
+            from components.java
+        }
+    }
+
+    repositories {
+        maven {
+            name = 'mt.edu.um.cf2.jgribx'
+            url = layout.buildDirectory.dir("repo")
+        }
+    }
+}
+
 sourceSets {
-	main {
-		resources.srcDirs = ["src/main/java","src/main/resources"]
-	}
-	test {
-		resources.srcDirs = ["src/test/java", "src/test/resources"]
-	}
+    main {
+        resources.srcDirs = ["src/main/java", "src/main/resources"]
+    }
+    test {
+        resources.srcDirs = ["src/test/java", "src/test/resources"]
+    }
 }
 
 configurations {
     extraLibs
```

Comment:
Can move this into the other configurations block so that we don't have multiple